### ISSUE TITLE
Tapping on menu items does nothing on a Glo HD running firmware 4.28

### ISSUE THREAD

**Issue by rtiangha** [Aryetis/Obenkyobo#1]
I installed the AIO 0.1.2 version on a Glo HD running firmware version 4.28.18220 and launched it using NickelMenu. The program loads successfully, but tapping on menu items does nothing. The options highlight (and stay highlighted when I remove my finger), but nothing gets activated. I can't even exit the program, so I have to hold down the power button until the device reboots. Here is the log.txt file:

```
Obenkyobo build: Oct 24 2021@16:45:29, cpp: 201703, Qt: 331523
Started at: 24-10-2021 13:44:20
kobofb: Finished initialization.
Kobo model: (alyssum,371), firmware: 3.14.0
LOG: MainWindow::MainWindow()::Start
"dhcpcd[2514]: dhcpcd not running\n"
"Selected interface 'eth0'\nOK\n"
LOG: MainWindow::MainWindow()::End
```

**Comment by Aryetis** [comment]
Thanks for the report,

According to the log, you're still running on the 3.14 firmware which is pretty much the oldest firmware possible for this device. NickelMenu requires a 4.6 firmware or newer to run correctly so you're pretty lucky that NM is even working at all. I recommend updating your firmware and giving it another try. 

To upgrade your firmware all you should have to do is :
1/ connect your device to the internet 
2/ go to Settings->Device Information->"Check for Update"

you can also upgrade it manually, to do so check those pages :
https://wiki.mobileread.com/wiki/Kobo_Firmware
https://wiki.mobileread.com/wiki/Kobo_Firmware_Releases

Edit : my bad I noticed you mentioned running on a 4.28 firmware. I ll double check the way I log firmware, it's probably wrong.

**Comment by Aryetis** [comment]
Can you try to install Ultimate Manga Reader / UMR and try to reproduce the bug ? https://www.mobileread.com/forums/showthread.php?t=331635  (mind the "Known issues" section, your touch inputs/screens are likely to be misoriented) 

As I am using a very similar setup and tools to develop my application. Therefore if the bug also happen with UMR the bug is likely caused by something in the Qt Setup / qt5-kobo-platform-plugin.

It's kinda hard to pinpoint the issue as I couldn't reproduce the issue on the 4 devices (non Glo HD) I tested so far.

**Comment by Aryetis** [comment]
Probably llinked to this issue 
https://github.com/Rain92/qt5-kobo-platform-plugin/issues/4

**Comment by Aryetis** [comment]
Sorry for the months long delay.
I've come with a solution that needs a bit more testing on various devices before pushing into the public. To verify that I didn't broke stuff touch inputs for other devices.

Anyways, here's the solution

1. Just to be sure, Install the new version of Obenkyobo (0.2.1)
2. Unzip this file and try replacing the old libkobo_obenkyobo.so file with this new one at : `.adds/qt-linux-5.15-kde-kobo/plugins/platforms`
[libkobo_obenkyobo.zip](https://github.com/Aryetis/Obenkyobo/files/8794556/libkobo_obenkyobo.zip) (built from https://github.com/Aryetis/qt5-kobo-platform-plugin/tree/df30d1cea23c4e02476bcfe37e56bccdecf576e3)

I'm guessing UltimateMangaReader is broken too atm because of this (don't try to replace UMR libkobo.so with this one, it won't work).

**Comment by Aryetis** [comment]
I finished testing my solution and consider it good enough to be shipped.
So there you go, it should be included in Obenkyobo v0.2.2

Please reopen this if the issue still occurs.

Fixed (and merged) with : 
https://github.com/Rain92/qt5-kobo-platform-plugin/commit/19118f8385b764b1c0e93c351f018d46dedf1d0e
